core/domain/topic_jobs_one_off.py

```diff
@@ -165,3 +167,80 @@ def reduce(key, values):
                 sum(ast.literal_eval(v) for v in values))])
         else:
             yield (key, values)
+
+
+class SubTopicPageMathRteAuditOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that checks for existence of math components in the SubTopicPage."""
+
+    _LATEX_STRINGS_WITHOUT_SVG = 'latex-strings-without-svg'
+    _LATEX_STRINGS_HAVING_SVG = 'latex-strings-having-svg'
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [topic_models.SubtopicPageModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+        subtopic = subtopic_page_services.get_subtopic_page_from_model(item)
+        html_string = ''
+        html_string += subtopic.page_contents.subtitled_html.html
+        html_string += ''.join(
+            subtopic.page_contents.written_translations.
```

Comment:
Just to check, none of the other models had  'written_translations'?

Reply:
No, i don't think so

Reply:
Well, all written translations are the same, right? So, they have html fields (the function misses that, it should ideally return it as well, can you maybe edit it?), but we don't have any way to record translations for skills now, so there won't be any html in the datastore, so I think it should be fine.